README.md

```diff
@@ -1,29 +1,49 @@
 <p align="center">
   <a href="https://vendia.net/">
-    <img src="https://raw.githubusercontent.com/vendia/examples/main/vendia-logo.png" alt="vendia logo" width="100px">
+    <img src="https://www.vendia.net/images/logo/black.svg" alt="vendia logo" width="250px">
   </a>
 </p>
 
 # Vendia Examples
+This repository contains a variety of examples that highlight the capabilities of [Vendia Share](https://www.vendia.net/product).
 
-A set of example applications that demonstrate how data can be ingested, queried, and manipulated using [Vendia products](https://vendia.net/product).
+## Common Prerequisites
+Each example will have its own set of detailed prerequisites.  Two prereqs that are almost always needed (so worth getting now) are:
 
-<details>
-<summary>Click for more detail</summary>
+* A Vendia Share [Account](https://share.vendia.net/signup), used to access Vendia Share
+* The Vendia Share [Command Line Interface (CLI)](https://vendia.net/docs/share/cli), used to manage Unis on Vendia Share
 
-- [Sign Up for Vendia](https://share.vendia.net/)
-- [Getting Started with Vendia](https://vendia.net/docs/share/quickstart)
-- [Product Information](https://vendia.net/product)
-- [Blog](https://vendia.net/blog)
-- [Contributing](https://github.com/vendia/examples/CONTRIBUTING.md)
-</details>
+## Overview
+The top-level directory structure groups examples by type for ease of navigation.
 
+| Directory | Purpose |
+|:---------|:--------|
+| [/applications](applications/README.md) | _Coming Soon!_ Full-stack sample applications, powered by Vendia Share.
+| [/approaches](approaches/README.md) | Architectural, design, and tooling approaches that demonstrate  ways to interact with and successfully leverage Vendia Share
+| [/demos](demos/README.md) | Guided, step-by-step industry and use case specific demonstrations using Vendia Share  
+| [/features](features/README.md) | Deeper exploration of the core features that comprise Vendia Share
+| [/integrations](integrations/README.md) | Deeper exploration of common ingress and egress sources to get data in to and out of Vendia Share
+| [/workshops](workshops/README.md) | Guided, step-by-step industry and use case specific workshops to educate those new to Vendia Share
```

Comment:
What's the thinking behind the bifurcation of use cases between demos vs workshops? What might go in one folder vs the other?  Is one more follow-along step-by-step?

Reply:
Good question.  The Workshops steps are meant to help get someone brand new to Share to work through a series of milestones, with the primary objective being learning more about Share.  The Demo steps are meant to help someone get a demo up and running, with the primary objective being seeing what Share can do.